egs/ljspeech/TTS/vits/tokenizer.py

```diff
@@ -80,13 +83,11 @@ def texts_to_token_ids(
 
             token_ids = []
             for t in tokens:
-                if t in self.token2id:
-                    token_ids.append(self.token2id[t])
-                else:
-                    token_ids.append(self.oov_id)
+                assert t in self.token2id, t
```

Comment:
```suggestion
                if t not in self.token2id:
                  logging.warning(f'Skip oov {t}')
                  continue
```

We just skip OOVs instead of throwing an assertion error, which
may kill the process.